### PR TITLE
adding_models_unittesting

### PR DESCRIPTION
**Pull request by musabjolani**
just for testing pull request..

### CODE REVIEW

**Comment by MohammadMouadi** [comment]
You could actually use it. if by writing "for testing" you mean that you will not use that.

File: myapp/urls.py [hunk 1]
```diff
@@ -9,4 +9,5 @@
 	path('accounts/login/', auth_views.LoginView.as_view(template_name='login.html'), name='login'),
 	path('accounts/logout/', views.logout_view, name='logout'),
 	path('accounts/register/', views.register, name='register'),
+	path('accounts/register/', views.register, name='register'),
```
Comment (ilanjb) [inline myapp/urls.py:12]: if you really need to push a test, better, to just add a comment...

**Comment by MohammadMouadi** [comment]
I notice there is a database file committed to your pull request, you should remove it from VCS and commit the changes, as it is not related to the pull request

File: myapp/tests.py [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+        self.assertIsNotNone(stock.price)
+
+    def test_change_percent_not_none(self):
+        stock = Stock.objects.get(symbol='1')
+        self.assertIsNotNone(stock.change_percent)
+
+    def test_primary_excahnge_length(self):
+        stock = Stock.objects.get(symbol='1')
+        max_length = stock._meta.get_field('primary_exchange').max_length
+        self.assertEquals(max_length, 32)
```
[the remark this answers is not in the document]
Comment (AlaaYa7ia) [reply myapp/tests.py]: I went over your tests. looks good. 
Fix the PR please so we can approve it :)

**Review by MohammadMouadi — APPROVED** [review]

**Pull request merged**